packages/@lwc/integration-karma/.eslintrc

```diff
@@ -2,7 +2,8 @@
     "globals": {
         "process": true,
         "LWC": true,
-        "spyOnAllFunctions": true
+        "spyOnAllFunctions": true,
+        "TestUtils": true
```

Comment:
I couldn't figure out how to `import` from `'test-utils'` in the hydration tests, but this works.